### PR TITLE
Message styling

### PR DESCRIPTION
**Pull request by cweems**
Changes the layout of the message bubbles so that they are the width of the text that they contain (rather than being fixed width). On mobile, message bubbles will be 100% the width of the container.

Also removes the use of flexbox CSS, making the messages view compatible with older browsers. Small but important fix.

screen shot 2017-07-10 at 1 39 18 pm
screen shot 2017-07-10 at 1 39 34 pm

### CODE REVIEW

File: app/assets/stylesheets/templates/_messages.scss [hunk 1]
```diff
@@ -28,23 +26,27 @@
...
-        border-radius: 8px;
+        border-radius: 10px 10px 0px 10px;
         color: $color-background;
         margin-bottom: 10px;
         padding: {
           bottom: .5em;
-          left: $padding-small;
-          right: $padding-small;
+          left: 1.5em;
+          right: 1.5em;
```
Comment (tmaybe) [inline app/assets/stylesheets/templates/_messages.scss]: Could you assign a variable for this value (`1.5em` or an equivalent in `rem`) in `app/assets/stylesheets/atoms/_variables.scss` and use it above, and on lines 74-75?

**Review by tmaybe — CHANGES_REQUESTED** [review]
Looks great to me! I made one small change request, but aside from that, if Manya's good with the visual/UX, I think it's good to go.

File: app/assets/stylesheets/templates/_messages.scss (hunk 1 above)
Comment (alsomanya) [reply app/assets/stylesheets/templates/_messages.scss]: In the screenshots the padding looks a little bigger than what's currently on production. is 1.5em the same amount?

**Comment by cweems** [comment]
@tmaybe This should be good to go!

**Review by tmaybe — APPROVED** [review]
Looks great! merging.

**Pull request merged**